### DISCUSSION TITLE
Prefix agregate view

### DISCUSSION THREAD

**Discussion by akashha** [netbox-community/netbox#5572] [Q&A] [answered]
Hello,

In the new version of netbox, the prefixes are automatically organized by aggregates.
I would like to know if it is possible to disable aggregation in the graphical interface ?
I am now drowning in my 6 prefix pages ..

Cordially.

**Comment by ryanmerolle** [comment, marked as answer]
Since 2.11 or 3.0 is the view any better if I recall some tweaks were made.  There is not currently a way to disable aggregation.
